metapackages/auto-configuration-propagators/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenTelemetry Meta Package for Propagators Configuration
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-url]
+
+## About
+
+This module provides a way to get a propagator based on the [OTEL_PROPAGATORS environment variable](env-var-url).
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/auto-configuration-propagators
+```
+
+## Usage
+
+Here is an example of how to retrieve a propagator:
+
+```js
+import { getPropagator } from '@opentelemetry/auto-configuration-propagators';
+
+const propagator = getPropagator();
+```
+
+Please see the [Supported propagators](#supported-propagators) section for more information.
+
+## Supported propagators
+
```

Comment:
Can we put a note here that the only propagators that will be supported are the ones from the spec? 
I'd like to refer to this in the future if someone wants to add additional ones that don't have a well-known value and might clash with future additions to the spec (this is something we don't have do deal with with instrumentations/resource detectors as they're configured unspecified `OTEL_NODE*` env vars).

Reply:
Added the note, please resolve if it looks good to you.

Reply:
looks good, thanks :+1: